packages/workflows-sdk/src/utils/composer/create-step.ts

```diff
@@ -121,6 +121,9 @@ function applyStep<
 
     const handler = {
       invoke: async (transactionContext) => {
+        transactionContext.context.idempotencyKey =
+          transactionContext.idempotency_key
```

Comment:
Shouldn't it be?
```
transactionContext.metadata.idempotency_key
```

Reply:
It is in too many places, but it is also in the transaction context 😅